src/WallpaperEngine/Render/CVideo.cpp

```diff
@@ -0,0 +1,30 @@
+#include "CVideo.h"
+
+using namespace WallpaperEngine;
+
+using namespace WallpaperEngine::Render;
+
+CVideo::CVideo (Core::CVideo* video, irr::video::IVideoDriver* driver) :
```

Comment:
When you want to have access to any part of the irrlicht engine it's better to pass Irrlicht::CContext* instead so you can have access to everything in case of needing it, for example, the video driver is accessible on context->getDevice ()->getVideoDriver ()